enhancements/testdata-separation.md

```diff
@@ -0,0 +1,109 @@
+---
+title: testdata-separation
+authors:
+  - "@theishshah"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-07-12
+last-updated: 2021-07-20
+status: TBD
+---
+
+# testdata-separation
+
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+The current bundle includes both an operator as well as its testdata. This results in inflated binaries and scenarios where users may want to release an operator bundle sans testdata but are unable to do so. The solution to this issue is to provide users the ability to use and generate a slim bundle which is decoupled from the testing necessities. Due to the nature of the testdata, it needs the rest of the bundle so it has limited value alone.
+
+## Motivation
+
+The motivation for this EP is to provide users with the option to deliver a slim bundle without testdata.
```

Comment:
What do you mean with testdata? 
Are you speaking about the dir `[tests](https://github.com/operator-framework/operator-sdk/tree/master/testdata/go/v3/memcached-operator/bundle/tests)/scorecard/` ? Because we have not a testdata dir in the bundle. 

Is the goal remove the file : https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/bundle/tests/scorecard/config.yaml? 

If yes, what is the size of this file? How much the authors will win with? 
Also, can we not only recommend dor authors delete the dir if they wish?